script/down

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
-CLUSTER="example"
-doctl k8s cluster delete ${CLUSTER}
\ No newline at end of file
+CLUSTER="kb-example"
```

Comment:
Would you mind keeping the name to `example`? That way, we stay consistent with the description of the manual steps in the main README.